deploy/__main__.py

```diff
@@ -0,0 +1,37 @@
+import pulumi
+import pulumi_digitalocean as do
+import pulumi_tls as tls
+import pulumi_random as random
+import rails_secret_key_base as rails
+
+config = pulumi.Config()
+
+with open('bin/in-droplet/cloud-init', 'r') as cloud_init_file:
+    cloud_init_script = cloud_init_file.read()
+
+with open('user_data', 'r') as user_data_file:
+    user_data = user_data_file.read() \
+        .replace("%cloud-init-script%", cloud_init_script)
+
+postgres_password = random.RandomPassword("cobra-postgres-password", length=16, special=False)
+rails_secret_key_base = rails.RailsSecretKeyBase("cobra-key-base")
+
+private_key = tls.PrivateKey("cobra-key", algorithm="RSA")
+ssh_key = do.SshKey("cobra-ssh-key", public_key=private_key.public_key_openssh)
+
+droplet = do.Droplet(
+    resource_name="cobra",
+    image="ubuntu-22-04-x64",
+    region=config.get("region", "lon1"),
+    size=config.get("size", "s-1vcpu-1gb"),
```

Comment:
make this a flag?

Reply:
It is, it's retrieving it from the Pulumi configuration with a default value.

Reply:
oh! i was looking for a literal python flag in this file and clearly not reading closely enough.  :D